source/blog/2016-04-06-naming-is-hard-or-who-let-the-vowels-out.html.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Naming is hard or Who Let The Vowels Out
+author: apevec
+tags: openstack, trunk, packaging
+date: 2016-04-06 16:54:12 CEST
+---
+
+
+When [Derek Higgins](http://www.openstack.org/blog/2014/02/open-mic-spotlight-derek-higgins/) has [started](https://github.com/openstack-packages/DLRN/commit/496baa4eb177873ba1421c6fbd7cc7041bd584a4) the tool to build OpenStack packages based on latest upstream aka "trunk" changes, name Delorean seemed like a good choice: tool is keeping older builds so you could "go back in time" like with a [time machine](https://en.wikipedia.org/wiki/DeLorean_time_machine), it's geeky enough and there's [Irish connection](https://en.wikipedia.org/wiki/DeLorean_Motor_Company#Manufacturing_facility).
```

Comment:
The history looks fine, a couple of small suggestion, I think make it read better

When Derek Higgins started   (leave out has I think)
**the** name Delorean seem.... 
**the** tool is keeping...